src/components/frame/LandingPage/LandingPage.tsx

```diff
@@ -1,125 +1,169 @@
+    defaultMessage:
+      'Define your identity, track your contributions, build your reputation.',
   },
-  createColony: {
-    id: `${displayName}.createColony`,
-    defaultMessage: 'Create a colony',
+  viewUserProfileButtonText: {
+    id: `${displayName}.viewUserProfileButtonText`,
+    defaultMessage: 'View',
   },
-  exploreColony: {
-    id: `${displayName}.exploreColony`,
-    defaultMessage: 'Explore the {colonyName}',
+  exploreMetacolonyTitle: {
+    id: `${displayName}.exploreMetacolony`,
+    defaultMessage: 'Explore the Metacolony',
   },
-  createUsername: {
-    id: `${displayName}.createUsername`,
-    defaultMessage: 'Create a username',
+  exploreMetacolonyDescription: {
+    id: `${displayName}.exploreMetacolonyDescription`,
+    defaultMessage: 'The Colony using Colony to build Colony.',
+  },
+  exploreMetacolonyButtonText: {
+    id: `${displayName}.exploreMetacolonyButtonText`,
+    defaultMessage: 'Explore',
+  },
+  inviteBlockTitle: {
+    id: `${displayName}.inviteBlockTitle`,
+    defaultMessage: 'Invite 1 person to create a Colony',
+  },
+  inviteBlockDescription: {
+    id: `${displayName}.inviteBlockDescription`,
+    defaultMessage:
+      'You can invite only one member to create a colony of their own using the new app during the private beta with this custom invite link: app.colony.io/createcolony/{invitationCode}',
   },
 });
 
-interface LandingItemProps {
-  to: string;
-  message: MessageDescriptor;
-  onClick?: () => void;
-}
-
-const LandingItem = ({ to, message, onClick }: LandingItemProps) => (
-  <li className={styles.item}>
-    <NavLink to={to} onClick={onClick} className={styles.itemLink}>
-      <Icon className={styles.itemIcon} name="circle-plus" title={message} />
-      <span className={styles.itemTitle}>
-        <FormattedMessage {...message} />
-      </span>
-    </NavLink>
-  </li>
-);
-
 const LandingPage = () => {
-  /*
-   * Are the network contract deployed to the chain the user is connected
-   * so that they can create a new colony on it
-   */
-  const { wallet, updateUser, user, userLoading } = useAppContext();
-  const canInteractWithNetwork = useCanInteractWithNetwork();
-  const { data, loading } = useGetMetacolonyQuery();
+  const [hoveredItem, setHoveredItem] = useState<number>(0);
+  const navigate = useNavigate();
+  const { user, connectWallet, wallet, userLoading } = useAppContext();
+  const onUserLogin = () => {
+    clearLastWallet();
+    connectWallet?.();
+  };
 
-  const [metacolony] = data?.getColonyByType?.items || [];
-
-  /* Ensures username is up-to-date post create user flow. */
-  useEffect(() => {
-    if (updateUser) {
-      updateUser(wallet?.address, true);
-    }
-  }, [wallet, updateUser]);
+  const landingPageItems = [
+    {
+      buttonText: MSG.createColonyButtonText,
+      headingText: MSG.createColonyTitle,
+      headingDescription: MSG.createColonyDescription,
+      iconName: 'layout',
+      // @TODO: Connect with real invitation code
```

Comment:
This box should now be disabled the only way for someone to create a Colony during the private beta is with a direct invite link otherwise it will get far too :spaghetti: